testdata/extension.config.yml

```diff
@@ -1,5 +1,5 @@
 port: 8000
-public_url: https://example.ngrok.io
+public_url: https://7cbc-2600-1700-5658-3c50-fdd0-8885-93fc-9d51.ngrok.io
```

Comment:
I guess it's time to remove this 😎 We can tophat it using `shopify extension serve` instead.

Reply:
Hmm... I don't think I understand what you are suggesting here.

Reply:
Let's restore this to `https://example.ngrok.io` because no one will be able to use my ngrok URL.